src/screens/EventVolunteers/VolunteerContainer.spec.tsx

```
@@ -46,20 +47,42 @@ vi.mock('react-router-dom', async () => {
+              <LocalizationProvider>
+                <I18nextProvider i18n={i18n}>
+                  <VolunteerContainer />
+                </I18nextProvider>
+              </LocalizationProvider>
+            </Provider>
+          </MemoryRouter>
+        </MockedProvider>
+      </TestErrorBoundary>,
+    );
+  });
+  return result!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)